### ISSUE TITLE
Max-Mahalanobis Training

### ISSUE THREAD

**Issue by P2333** [robust-ml/robust-ml.github.io#10]
**Name**: {Rethinking Softmax Cross-Entropy Loss for Adversarial Robustness }

**Authors**: {Tianyu Pang, Kun Xu, Yinpeng Dong, Chao Du, Ning Chen, Jun Zhu}

**Paper**: {https://arxiv.org/pdf/1905.10626.pdf}

**Code**: { https://github.com/P2333/Max-Mahalanobis-Training }

**Venue**: {venue if published}

**Does the code implement the robust-ml API and include pre-trained models**: {no}

**Dataset**: {cifar10}

**Threat model**: {ℓ∞(ϵ=8/255)}

**Natural accuracy**: {93.6%}

**Claims**: {46.0%}
